test/testdata/namer/alias_cross_file.flattened-tree.exp

```diff
@@ -1,5 +1,12 @@
 begin
-  ::Sorbet::Private::Static.keep_for_ide(::TestSig)
+  class <emptyTree><<C <root>>> < ()
+    def self.<static-init><<static-init>$CENSORED>(<blk>)
+      begin
+        ::Sorbet::Private::Static.keep_for_ide(::TestSig)
```

Comment:
this seems more correct to have this in a `static-init` instead of just floating in an `InsSeq`